build.gradle

```diff
@@ -435,6 +447,7 @@ subprojects {
                 attributes("Implementation-Version": "${version}")
                 attributes("Implementation-URL": "https://checkerframework.org")
                 attributes('Automatic-Module-Name': "org.checkerframework." + project.name.replaceAll('-', '.'))
+                attributes('Export-Package': '*')
```

Comment:
I think the reason this does not work is here https://plugins.gradle.org/plugin/biz.aQute.bnd

I you go to the bottom and expand the "about new plugin" section it says:

> Cannot be used in conjunction with subprojects {}, allprojects {} etc.
> It is not possible to use the familiar pattern of applying a plugin to multiple projects at once using subprojects {} etc. at the moment. There is currently no mechanism for applying a plugin to multiple projects at once. At the moment, each project that requires a plugin must declare so in the plugins {} block in its buildscript.
> 
> Future versions of Gradle will remove this restriction.
> 
> If the current limitations of the new plugin mechanism are problematic for your build, the recommended approach is to use the “traditional” mechanism for the time being.

This seems to suggest to me that each of the individual sub project grade file need to declare this plugin them selves. Kinda unfortunate you cant centralize the config like this though.

Reply:
Thanks for looking that up. I'll abandon this for now until somebody actually requires this feature.